qt/scientific_interfaces/Muon/ALCDataLoadingPresenter.cpp

```diff
@@ -324,9 +243,9 @@ void ALCDataLoadingPresenter::load(const std::vector<std::string> &files) {
     emit dataChanged();
 
   } catch (const std::invalid_argument &e) {
-    m_view->displayError(e.what());
+    throw std::runtime_error(e.what());
```

Comment:
why are we throwing instead of displaying?

Reply:
The reason for this is that now there are some extra methods that need to be called on success or failure of load. Here is where the exception is caught and then the error displayed: https://github.com/mantidproject/mantid/blob/f955831bbfea204ad849269549c57cfbb18fbafd/qt/scientific_interfaces/Muon/ALCDataLoadingPresenter.cpp#L114
If the error was displayed in load I would have to move the code in the catch statement in this try into the load method itself, or somehow indicate load has failed such as returning a bool

Reply:
can you add a line here saying it is caught in <method name>

In the try can you add a comment saying it catches from load method

Then we will have some protection from it being deleted by accident